jetty-util/src/main/java/org/eclipse/jetty/util/TopologicalSort.java

```diff
@@ -49,7 +45,7 @@
  * 
  * @param <T> The type to be sorted. It must be able to be added to a {@link HashSet}
  */
-public class TopologicalSort<T> implements Dumpable
```

Comment:
There's nothing saying we can't just make a DumpableTopologicalSort extended version with this support for use within Jetty proper.

Reply:
toString has all the details needed. Dumpable is just a bit better formatted.